aces/Loris
# [Statistics / Behavioural] fix query by project.

## Brief summary of changes

- fixes some bugs found in the code.
- adds permission restriction over the projects the user is able to query 

The query by project should be working now for `Reports->Statistics->Behavioural Statistics`

#### Testing instructions (if applicable)

1. Go to `Reports->Statistics->Behavioural Statistics`
2. Select different projects on the `Projects ComboBox ` and click on `Submit Query`
3. The properly information should be retrieved.

#### Link to related issue

* Resolves #6573
